docs/src/pages/docs/rating.tsx

```diff
@@ -88,15 +83,23 @@ const RatingPage: React.FC = (): React.ReactElement<void> => {
     const importString: string = React.useMemo(() => require("!raw-loader!@sebgroup/react-components/Rating/Rating"), []);
     const importedFiles: Array<string> = React.useMemo(() => [require("!raw-loader!@sebgroup/react-components/Rating/Rating")], []);
     const code: string = `
-        <Rating 
-            initialValue={number}
-            useHollow={boolean}
-            readOnly={boolean}
-            disabled={boolean}
-            iconHeight={number}
-            iconWidth={number}
-            colors={Array<string>}
-        />`;
+    <Rating
+        id="some-id"
+        className="custom-class"
+        readOnly={boolean}
+        disabled={boolean}
+        dimension={React.ReactText}
+        max={React.ReactText}
+        min={React.ReactText}
+        step={React.ReactText}
+        svgname={string}
+        colors={[string, string]}
+        customSVG={JSX.Element}
+        value={number}
+        onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+            setValue(Number(e.target.value));
+        }}
+    />`;
```

Comment:
The example should resemble the minimum amount of code needed to get it up and running.
```suggestion
    <Rating value={number} onChange={(e) => setValue(e.target.value)} />`;
```